src/selectedtests/helpers.py

```diff
@@ -28,7 +28,7 @@ def get_mongo_wrapper() -> MongoWrapper:
     """
     mongo_uri = os.environ.get("SELECTED_TESTS_MONGO_URI")
     if mongo_uri is None:
-        raise RuntimeError("Cannot connect to mongodb, SELECTED_TESTS_MONGO_URI is not set")
+        raise RuntimeError("Cannot connect to mongodb, SELECTED_TESTS_MONGO_URI is not set looooooooooong word")
```

Comment:
?

Reply:
Nvm, you mentioned reducing the line length so this is to trip a failure I think.

Reply:
see git commit message for explanation :p